index.html

```diff
@@ -0,0 +1,70 @@
+<!DOCTYPE html>
+<html lang="en">
+
```

Comment:
- [ ] Please make sure to add meta information relevant for passing linter errors.